### PR TITLE
 Make pulumi architecture more discoverable

### PR DESCRIPTION
**Pull request by dixler**
Addresses #1461

### CODE REVIEW

**Comment by susanev** [comment]
@dixler can you add a redirect?
https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md#links

**Comment by dixler** [comment]
@susanev I don't believe I changed the URL. I just renamed the text of the links. Does this still apply?

**Comment by dixler** [comment]
will fix
2022-05-04-121503_876x632_scrot

**Comment by dixler** [comment]
@susanev well now I'm aware of the changing urls

**Review by susanev — APPROVED** [review]
@dixler a few questions..
- should we update this text `Learn about the Pulumi architecture, core concepts and programming model.` on the intro landing page to mention "how pulumi works"?
- should we move how pulumi works to the top of the list? or does it still make sense at the bottom?

File: themes/default/content/docs/intro/concepts/_index.md [hunk 1]
```diff
@@ -269,58 +269,58 @@ outputs:
 The following topics provide more details on the core concepts of Pulumi and how to use it:
 
 <div class="md:flex flex-row mt-6 mb-6">
+    <div class="md:w-1/2 border-solid md:ml-4 border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts/how-pulumi-works" >}}"><i class="fas fa-upload pr-2"></i>How Pulumi Works</a></h3>
+        <p>Learn about how Pulumi performs deployments under the hood.</p>
```
Comment (dixler) [inline themes/default/content/docs/intro/concepts/_index.md:274]: @susanev fixed the grammar

**Pull request merged**